core/templates/dev/head/components/answer_group_editor_directive.html

```diff
@@ -50,4 +50,47 @@
                     suppress-warnings="suppressWarnings()">
     </outcome-editor>
   </div>
+  <style>
+    .oppia-rule-header {
```

Comment:
Now that these rules are tied to a class, can we do some renaming? Can we write something like:

    answer-group .oppia-rule-header

so that everything here is scoped to answer-group tags only? That will prevent collisions with future DOM elements if someone ever wants to add an .oppia-rule-header class somewhere else in the app in the future.

Ditto for everything else.